tonic/src/status.rs

```diff
@@ -325,7 +325,10 @@ impl Status {
         })
     }
 
-    pub(crate) fn try_from_error(
+    /// Create a `Status` from various types of `Error`.
+    ///
+    /// Returns the error if a status could not be created.
```

Comment:
```suggestion
    /// Create a `Status` from various types of `Error`.
    ///
    /// Returns the error if a status could not be created.
    ///
    /// # Downcast stability
    /// This function does not provide any stability guarantees around how it will downcast errors into
    /// status codes.
```